### PR TITLE
FLW-1491 new account icons

### PR DESCRIPTION
**Pull request by Deneath**

### CODE REVIEW

**Review by arvifox — APPROVED** [review]

File: fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt [hunk 1]
```diff
@@ -2,27 +2,40 @@ package jp.co.soramitsu.fearless_utils.icon
...
+
         private val SCHEMES = mutableListOf(
-            Scheme("target", 1, arrayOf(0, 28, 0, 0, 28, 0, 0, 28, 0, 0, 28, 0, 0, 28, 0, 0, 28, 0, 1)),
+            Scheme(
+                "target",
+                1,
+                arrayOf(0, 28, 0, 0, 28, 0, 0, 28, 0, 0, 28, 0, 0, 28, 0, 0, 28, 0, 1)
+            ),
             Scheme("cube", 20, arrayOf(0, 1, 3, 2, 4, 3, 0, 1, 3, 2, 4, 3, 0, 1, 3, 2, 4, 3, 5)),
```
Comment (PankraSerg) [inline fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt:21]: fold as before - better readability

File: fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt [hunk 2]
```diff
@@ -2,27 +2,40 @@ package jp.co.soramitsu.fearless_utils.icon
...
+                "vmirror",
+                128,
+                arrayOf(0, 1, 2, 3, 4, 5, 3, 4, 2, 0, 1, 6, 7, 8, 9, 7, 8, 6, 10)
+            ),
+            Scheme(
+                "hmirror",
+                128,
+                arrayOf(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 8, 6, 7, 5, 3, 4, 2, 11)
+            )
         )
```
Comment (PankraSerg) [inline fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt:35]: same, fold

File: fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt [hunk 3]
```diff
@@ -68,7 +108,8 @@ class IconGenerator {
         val totalFreq = SCHEMES.fold(0) { summ, schema -> summ + schema.frequency }
 
         val zeroHash = Blake2b.Blake2b512().digest(ByteArray(32) { 0 })
-        val idHash = Blake2b.Blake2b512().digest(id).mapIndexed { index, byte -> (byte + 256 - zeroHash[index]) % 256 }
+        val idHash = Blake2b.Blake2b512().digest(id)
+            .mapIndexed { index, byte -> (byte + 256 - zeroHash[index]) % 256 }
```
Comment (PankraSerg) [inline fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt:113]: unnecessary, fold

File: fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt [hunk 4]
```diff
@@ -95,7 +136,8 @@ class IconGenerator {
         }
 
         val rot = (idHash[28] % 6) * 3
-        val colors = scheme.colors.mapIndexed { index, _ -> palette[scheme.colors[if (index < 18) (index + rot) % 18 else 18]] }
+        val colors =
+            scheme.colors.mapIndexed { index, _ -> palette[scheme.colors[if (index < 18) (index + rot) % 18 else 18]] }
```
Comment (PankraSerg) [inline fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/icon/IconGenerator.kt:140]: same, fold

**Pull request merged**